mungegithub/github/github.go

```diff
@@ -48,6 +49,11 @@ const (
 	headerRateReset     = "X-RateLimit-Reset"
 )
 
+var (
+	releaseMilestoneRE = regexp.MustCompile(`^v[\d]+.[\d]$`)
```

Comment:
It is the name of the milestone it will match again, so in kubernetes it will today only (likely) see 3 possible values:
`v1.2` (matches), `v1.2-candidate` (will not match), `next-candidate` (will not match).  (It could actually get closed milestones, but that won't happen much/ever)